views/install.sh.erb

```diff
@@ -290,7 +292,7 @@ capture_tmp_stderr() {
 # do_wget URL FILENAME
 do_wget() {
   echo "trying wget..."
-  wget -O "$2" "$1" 2>$tmp_dir/stderr
+  wget -t 1 -T $download_timeout -O "$2" "$1" 2>$tmp_dir/stderr
```

Comment:
@lamont-granquist You're right that if you're offline then it will fail near instantly.  I added timeouts to cover those edge cases where you have internet but nothing downloads all the same, like a misconfigured proxy.

According to the [GNU docs on wget](http://www.gnu.org/software/wget/manual/html_node/Download-Options.html#index-timeout), there are three different timeouts: `--dns-timeout`, `--connect-timeout`, and `--read-timeout`; with `-T` setting all of them.  
I think setting all of them is reasonable (as `--read-timeout` only timeouts if the connection is idle for the specified amount of time), especially after the `download_timeout` gets bumped to something much higher.